api/libproject/devstart.rb

```diff
@@ -2035,7 +1935,7 @@ def delete_runtimes(cmd_name, *args)
   api_url = get_leo_api_url(gcc.project)
   ServiceAccountContext.new(gcc.project).run do
     common.run_inline %W{
-       gradle manageLeonardoRuntimes
+       ./gradlew manageLeonardoRuntimes
```

Comment:
AFAICT you caught all of these, as well as all `ensure_docker`s and `assert_in_docker`s 👍